scripts/build_gnome_supp.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Don't forget copyright headers in these files too.

Reply:
I'm not that much into `law`/`copyright` stuff,... Why does it matter?

Reply:
Funny way to make `+126` changes: https://github.com/ddccontrol/ddccontrol/pull/55/commits/e84074cc1c7abb9b0d8ca9006ade8dd2893c89d2

Reply:
You should not add the full boilerplate on each file, but just add an SPDX header.

Reply:
What's SPDX header?

Reply:
First matches when searching for "SPDX header"
https://lwn.net/Articles/739183/
https://github.com/david-a-wheeler/spdx-tutorial/blob/master/README.md

Reply:
@StefanBruens So, I would be:

```
# Copyright(c) 2018 Miroslav Kravec (kravec.miroslav@gmail.com)
#
# SPDX-License-Identifier: GPL-2.0
```

Correct?

Reply:
@StefanBruens I found it,.. way too long `README`: https://github.com/david-a-wheeler/spdx-tutorial/blob/master/README.md#spdx-license-expressions-in-source-code-files

Reply:
@StefanBruens anyway, thanks for info.